### ISSUE TITLE
Can you share the test code?

### ISSUE THREAD

**Issue by zxk72** [ibrahimgh25/EL-GAN-Implementation#3]
Thanks for your excellent work!
I read your other answer, but I am not very good at it. I would like to ask if you can send me the code of only the test part?
Looking forward to your reply, thank you very much!

**Comment by ibrahimgh25** [comment]
Hello,

Unfortunately I don't have the "test" code. I tried running the project, a lot has changed since the code was written, I can't even find pytorch 1.8.0 on pypi. In any case, if you were able to somehow download the project, the data, and run the training script you're already there. the testing would be predicting with the trained model (generator) and comparing your results with the ground truth, you could use the loss function (pixel-wise cross-entropy), or this much better evaluation script from the challenge: [evaluate.py](https://github.com/TuSimple/tusimple-benchmark/blob/master/evaluate/lane.py).

I might in the coming days/weeks work on updating the project to work with python 3.10 (I was working on 3.6 at the time I think), and pytorch 2.1, but for that I will have to rewrite a lot of the code, since it's messy. I might try training the model again, if I do I will let you know.

**Comment by zxk72** [comment]
@ibrahimgh25 
Thank you very much for your reply.
I did encounter some troubles running the existing code, and I am trying to figure it out. In the meantime, I look forward to your updates, and I am very grateful for your updates on the code, which are like treasures to me. Thank you!
